metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wazuh-wazuh",
-  "version": "2.0.22",
```

Comment:
Hi @rafaelfc-olx 
The variable version refers to the version of the puppet package for wazuh and not the wazuh-hids.

Reply:
thanks for the heads up. just reverted this specific part.